test/graphiteThreshold.check.spec.js

```diff
@@ -33,73 +32,124 @@ describe('Graphite Threshold Check', function(){
 		check.stop();
 	});
 
+	context('Upper threshold enforced', function () {
+
+		it('Should use maxSeries Graphite function to acquire metrics', function (done) {
+			mockGraphite([0]);
```

Comment:
doesn't look quite the same as the data format in the PR description; best to make it realistic